src/pages/account/account.js

```diff
@@ -0,0 +1,110 @@
+import * as nodeUtils from "../../utils/nodeUtils.js";
+// 정규 표현식
+const expEmail =
+  /^[0-9a-zA-Z]([-_.]?[0-9a-zA-Z])*@[0-9a-zA-Z]([-_.]?[0-9a-zA-Z])*.[a-zA-Z]{2,3}$/i;
+
+/***
+ *유효성 검사 함수
+ *
+ *
+ *1.Null check
+ *2.타입에 맞게 양식을 제대로 작성했는지 정규표현식 검사
+ *3.조건에 맞지 않을 시 에러 메시지 생성
+ *
+ */
+function validationForm(e) {
```

Comment:
```suggestion
function validateInput(e) {
```
전체적인 동작을 봤을 때 input의 값이 유효한지 확인하고 그에 따른 오류 메시지를 하단에 삽입하는 함수로 보이는데요, 동사로 함수명을 지어주면 어떨까요?

Reply:
확실히 동사로 지어놓는게 더 직관적인거 같습니다.